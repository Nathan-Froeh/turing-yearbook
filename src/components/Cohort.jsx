import React from 'react';
import Person from './Person';
import './Cohort.css';

// COHORT COMPONENT CODE GOES HERE

function Cohort({group, groupType}) {
  console.log('Render Cohort')
  const groupMembers = group.map(person => <Person person={person} key={person.id}/>)

  return (
    <main>
      <h1>{groupType}</h1>
      <section className='people'>
      {groupMembers}
      </section>
    </main>
  )
}


export default Cohort;