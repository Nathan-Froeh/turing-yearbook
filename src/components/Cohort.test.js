import React from 'react';
import Cohort from './Cohort';
import { shallow } from 'enzyme';


describe('Cohort', () => {

  it('should render all components in correct order', () => {
    const component = shallow(
      <Cohort group={[]} groupType={'Staff'}/>
    )
    expect(component).toMatchSnapshot()
  })

})