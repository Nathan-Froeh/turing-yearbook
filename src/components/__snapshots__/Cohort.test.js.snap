// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cohort should render all components in correct order 1`] = `ShallowWrapper {}`;
